hera_mc/cm_revisions.py

```diff
@@ -91,15 +94,15 @@ def get_last_revision(hpn, session=None):
         end_date = revisions[rev]['ended']
         if end_date is None:
             latest_end = cm_utils.get_astropytime('>')
-            no_end.append(Namespace(hpn=hpn, rev=rev, started=revisions[rev]['started'],
+            no_end.append(Namespace(hpn=hpn, rev=rev, rev_query=rq, started=revisions[rev]['started'],
```

Comment:
I also think it would be better just to write `rev_query='LAST'`, and avoid the variable, here and below. These functions aren't very long, but using the variable still puts unneeded distance between the actual value in question and its usage.